NuGet.config

```diff
@@ -8,7 +8,6 @@
     <clear />
     <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
```

Comment:
Should we also remove this one? I don't think there's anything we still expect to only be in the blob feed.